.github/workflows/docker-publish.yml

```diff
@@ -73,3 +73,29 @@ jobs:
           tag_with_sha: true
           repository: mprooms/nginx
           path: nginx
+
+  deploy:
+    needs: push
+    runs-on: ubuntu-latest
+
+    # Do not run this for PRs
+    if: github.event_name == 'push'
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Deploy via SSH
+        run: |
+          umask 0077
+          mkdir ~/.ssh
+          echo "${{ secrets.SSH_PRIVATE_KEY_MPROOMS_VM }}" > ~/.ssh/id_rsa
+          echo -e "Host mprooms-vm.iot.mp-labs.de\nUser ubuntu\nStrictHostKeyChecking=no" > ~/.ssh/config
```

Comment:
If you clone this repo, do you download files in .github too? If yes, do we want these hostnames to be in there or should they be injected via environment variables too? Same for the ssh command a few lines down

Reply:
True, fixed.